scripts/update/02-run.sh

```diff
@@ -12,4 +12,13 @@ echo "========= Stage: Post-update =========="
 echo "======================================="
 echo
 
-# Nothing here for now
+# Make Umbrel OS specific post-update changes
+if [[ ! -z "${UMBREL_OS:-}" ]]; then
```

Comment:
Ah okay, this could be the reason why `dphys-swapfile` wasn't removed locally on my Pi. I think we need to source `/etc/default/umbrel` first for `$UMBREL_OS` to be available.

Reply:
Yeah it is, I just tested, I just ninja added that line in before you tested.

Although it seems a bit pointless to source `/etc/default/umbrel` and then test for the existence of a variable from the file we just sourced, when we could just test for the file.

I think there should be somewhere we can source `/etc/default/umbrel` that makes it available everywhere.

Reply:
Sourcing the file for now in this PR but I think we should create `/etc/profile.d/umbrel-os.sh` with:

```shell
export UMBREL_OS=$version
```

Reply:
> Although it seems a bit pointless to source /etc/default/umbrel and then test for the existence of a variable from the file we just sourced, when we could just test for the file.

I agree, although the use-case right now is just testing for the variable, it can be testing for multiple variables as well as their values. For example, we can test if `$UMBREL_OS` is a specific version and then do something.

> Sourcing the file for now in this PR but I think we should create /etc/profile.d/umbrel-os.sh

That's a good idea. Are you sure the environment variables will be available for root though? Because we already source `/etc/default/umbrel` in `/etc/profile` but they're still not exposed to the root.

Reply:
That could be cause we're not exporting them you need to do `export ENV=value` to ensure the shell session exports the variable to any child processes.

But no I'm not sure about root, I need to test it.